definitions/ext-solarwinds_server/definition.yml

```diff
@@ -16,12 +16,11 @@ synthesis:
     npm.ManageBy:
     npm.SiteCode:
     npm.ServerLocation:
-    datacenter:
```

Comment:
Hi,

I just wanted to double check if this change is intended ?
It seems to be unrelated of the change described in the PR.

Thanks

Reply:
Yes it is intended, removed the "datacenter" tag as well.